docs/guide/configuration.rst

```diff
@@ -47,10 +47,21 @@ Joint Dynamics
 ~~~~~~~~~~~~~~
 Joint dynamics are configured using environment variables starting with ``JAXSIM_JOINT_``. Available variables include:
 
+- ``JAXSIM_JOINT_FRICTION_STATIC``: Overrides the static friction coefficient for all joints.
```

Comment:
Why the decision to put them as env variables?
 
These values should be dependent on the model. I am not convinced of having the default set via env variables.